packages/manager/src/managers/prismicRepository/types.ts

```diff
@@ -80,7 +80,7 @@ export type AllChangeTypes =
 	| CustomTypeInsertChange
 	| CustomTypeUpdateChange
 	| CustomTypeDeleteChange;
-export interface BulkBody extends Record<string, unknown> {
+export interface PushBody extends Record<string, unknown> {
```

Comment:
❓ #ask: Do we need the `extends Record<string, unknown>`?

Reply:
Haha clearly not necessary, good catch!

Reply:
Removed